Bomberfrau/src/main/java/uni/bombenstimmung/de/game/Bomb.java

```diff
@@ -309,6 +373,31 @@ public int getCounter() {
 	return this.timer-this.counter;
     }
     
+    public double getScale() {
+	return scale;
+    }
+    
+    public void setCounter(int c) {
+	this.counter = c;
+    }
+    
+    public Field getPlacedField() {
+	return placedField;
+    }
+    
+    public void chainreaction(int x, int y) {
+	ArrayList<Bomb> placed = Game.getPlacedBombs();
+	for (Bomb j : placed) {
```

Comment:
`for (Bomb j : placed) {`
Wieder bitte passende Variablennamen verwenden. z.B.
`for (Bomb bomb : placed) {`